packages/headless/src/features/insight-search/insight-search-actions-loader.ts

```diff
@@ -45,6 +46,20 @@ export interface InsightSearchActionCreators {
     AsyncThunkInsightOptions<StateNeededByExecuteSearch>
   >;
 
+  /**
+   * Creates an action that executes a search query to fetch a new page of results.
+   *
+   * @param analyticsSearchAction - The analytics action to log after a successful query. See `loadSearchAnalyticsActions` for possible values.
+   * @returns A dispatchable action.
+   */
+  fetchPage(
+    analyticsSearchAction: SearchAction
+  ): AsyncThunkAction<
+    ExecuteSearchThunkReturn,
+    SearchAction,
```

Comment:
for insight-actions we use `InsightAction` instead of `SearchAction`

Reply:
Ah every other action is SearchAction so I'll update the other ones too. Thanks